src/config/rules-axe-options.js

```diff
@@ -0,0 +1,17 @@
+const rulesAxeOptions = {
+  /**
+   * Rule: role attribute has valid value
+   * Url: https://act-rules.github.io/rules/674b10
+   *  - Disable below checks
+   *    - `fallbackrole`
+   *    - `invalidrole`
+   */
+  '674b10': {
+    checks: {
+      'fallbackrole': { enabled: false },
+      'invalidrole': { enabled: false }
```

Comment:
Needed for passing test case - https://act-rules.github.io/rules/674b10#passed-example-3